src/tracer.js

```diff
@@ -425,7 +425,7 @@ module.exports.sendTrace = function sendTrace(runnerUpdateFunc) {
  */
 module.exports.sendTraceSync = function sendTraceSync() {
     const tracerObj = module.exports.getTrace();
-    if (!tracerObj || tracerObj.disabled === true) {
+    if (!tracerObj || tracerObj.disabled) {
```

Comment:
This is not a good place to test, because if tracerObj is undefined you'll get an exception